aws-greengrass-testing-features/aws-greengrass-testing-features-localdeployment/src/main/resources/greengrass/components/recipes/local_hello_world_multiplatform.yaml

```diff
@@ -0,0 +1,32 @@
+#
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+
+---
+RecipeFormatVersion: 2020-01-25
+ComponentName: com.aws.LocalHelloWorldMultiplatform
+ComponentVersion: '1.0.0'
+ComponentDescription: Hello World Multiplatform Local Component.
+ComponentPublisher: Amazon
+Manifests:
+- Platform:
+    os: windows
+  Artifacts:
+    - URI: "file:C:/Windows/System32/cmd.exe"
+      Permission:
+        Read: ALL
+        Execute: ALL
+  Lifecycle:
+    run: |
+      cmd /c echo "Hello World!"
+- Platform:
+    os: linux
+  Artifacts:
+    - URI: file:/bin/bash
```

Comment:
It's more OS compatible to use /bin/sh. Just make sure you don't use any bash specific features.

Reply:
In that case we use bash/echo feature -e